docusaurus.config.js

```diff
@@ -107,7 +119,7 @@ const config = {
             {
               items: [
                 {
-                  html: `<a href="https://cosmos.network"><img src="/img/logo-bw.svg" alt="Cosmos Logo"></a>`,
+                  html: `<a href="https://cosmos.network"><img src="static/img/logo-bw.svg" alt="Cosmos Logo"></a>`,
```

Comment:
This should not be `/static`. It should stay the same. I was talking about the folder path in docusarus.